cfgov/ask_cfpb/locale/es/LC_MESSAGES/django.po

```diff
@@ -145,4 +145,4 @@ msgstr "Ver todos los resultados para"
 
 #: ask_cfpb/templates/admin/export.html:3
 msgid "Export Ask data"
-msgstr ""
+msgstr ""
```

Comment:
You could negate this change and keep the newline, which we generally do (which is what the 🛑 symbol is saying here)